package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-gelreportmodels",
-  "version": "0.7.0",
```

Comment:
does this version needs to be consistent with the other?

Reply:
I checked with @gerlv